src/main/java/seedu/jarvis/model/financetracker/Instalment.java

```diff
@@ -0,0 +1,46 @@
+package seedu.jarvis.model.financetracker;
+
+import java.util.Objects;
+
+/**
+ * Object stores a single recurring payment with its details such as description and the amount spent.
+ */
+public class Instalment {
```

Comment:
British or American English?

Reply:
Hello Jeffry, should it be in British English or American English?

Reply:
Consistency matters the most. The original code base is compliant to this : https://oss-generic.github.io/process/docs/FormatsAndConventions.html#english

Reply:
alright thank you!